torch/_inductor/optimize_indexing.py

```diff
@@ -10,7 +10,7 @@
 from .utils import dominated_nodes
 
 
-def val_expressable_in_32_bits(val):
+def val_expressable_in_32_bits(val: Any) -> bool:
```

Comment:
nit: we prefer `object` to `Any` bc `object` will do more type checking on the function. although i know in some cases its easier to get `Any` type checking and thats fine